config.js

```diff
@@ -0,0 +1,5 @@
+export default class Url{
```

Comment:
creating a class just to store single string seems like an overkill for me. I can just be stored in a dictionary in this config file